stdpopsim/cli.py

```diff
@@ -405,9 +433,20 @@ def run_simulation(args):
                 "populations")
         samples = model.get_samples(*args.samples)
 
-        contig = species.get_contig(
-            args.chromosome, genetic_map=args.genetic_map,
-            length_multiplier=args.length_multiplier)
+        if args.chromosome == "any":
```

Comment:
I don't see the need for randomly selecting a chromosome, seems like a complicated thing to implement. Are we choosing a chromosome randomly from those available? Or is it weighted by sequence lenght? Or is it weighted by overall recombination rate?

Reply:
It's weighted by sequence length. If a genetic map is provided, and the user wants just a specific length, then sampling a chromosome at random makes sense to me. I prefer this behaviour to using `--length-multiplier`, partly because it's just easier to choose the length.

Reply:
Yeah... it's not really coupled to the start, end functionality though, so I'd vote on pulling it out and doing in a follow up.